### PR TITLE
Add `Client.send_video` high-level method

### LINKED ISSUE #386: Add `Client.send_video` method
With video support coming to Bluesky very soon, I have code for a video-posting bot mostly ready to go. 

I noticed that while we have [`Client.send_image`](https://github.com/MarshalX/atproto/blob/e5c6a3bc0b8dbacb40fb634bac473782f24d4ed9/packages/atproto_client/client/client.py#L241) and [`Client.send_images`](https://github.com/MarshalX/atproto/blob/e5c6a3bc0b8dbacb40fb634bac473782f24d4ed9/packages/atproto_client/client/client.py#L189) for easy image uploading, there are no corresponding `Client.send_video` or `Client.send_videos` methods. It appears there is already code written for handling video uploading on a lower level of the API, and it just needs to have the higher-level Client wrappers written for it. 

This might be something I could try my hand at, although it wouldn't surprise me that others are already working on it and/or are much more experienced with the codebase than I am (that is to say, they have experience with the codebase, haha).

### PR DESCRIPTION
**Pull request by Meorge**
closes #386. This PR adds a method, `send_video`, to the `Client` class that behaves very similarly to `Client.send_image`, allowing the user to easily upload a post with a video attached.

### CODE REVIEW

File: packages/atproto_client/client/client.py [hunk 1]
```diff
@@ -278,6 +278,50 @@ def send_image(
...
+        if video_alt is None:
+            video_alt = ""
+
+        upload = self.upload_blob(video)
+
+        return self.send_post(
+            text,
+            profile_identify=profile_identify,
+            reply_to=reply_to,
+            embed=models.AppBskyEmbedVideo.Main(video=upload.blob, alt=video_alt),
```
Comment (naurojunior) [inline packages/atproto_client/client/client.py:320]: Nice PR!

Does it make sense to update the `send_post` method signature to accept `'models.AppBskyEmbedVideo.Main' too? 

Here: https://github.com/MarshalX/atproto/blob/main/packages/atproto_client/client/client.py#L118-L122

What do you think?

Comment (Meorge) [reply packages/atproto_client/client/client.py:320]: That makes sense! Will try to get that added and pushed soon

**Comment by MarshalX** [comment]
Thank you! Please do not forget to provide async version too. Typically you just need to run client code gen. And please run ruff over new code

**Review by MarshalX — COMMENTED** [review]
Thank you! Please do not forget to provide async version too. Typically you just need to run client code gen. And please run ruff over new code

**Comment by MarshalX** [comment]
@Meorge I fix it. async gen was a tricky wan because of some hacks around `upload_blob` usage. Btw to run gen you can use `poetry run atp gen async`.

So the last question is do we want to do something with captions? Yes, the image has only alt, but video alt and captions. Do we want to add args for captions here?

I do not understand why "captions" is the array of files... do they expect some captions file format or just txt with timestamp - text...

**Review by MarshalX — APPROVED** [review]

**Comment by Meorge** [comment]
OOPS TIME TO CHANGE MY PASSWORD

**Comment by MarshalX** [comment]
thank you for your contribution!

**Comment by Meorge** [comment]
No problem, thank you for testing and accepting it! 😄

**Pull request merged**